modules/ROOT/pages/clauses/remove.adoc

```diff
@@ -66,6 +68,42 @@ Properties set: 1
 `REMOVE` cannot be used to remove all existing properties from a node or relationship.
 Instead, using xref::clauses/set.adoc#set-remove-properties-using-empty-map[`SET` with `=` and an empty map as the right operand] will clear all properties from the node or relationship.
 
+[role=label--new-5.24]
+[[remove-remove-a-property-dynamically]]
+== Dynamically removing a property
+
+`REMOVE` can be used to remove a property on a node or relationship even when the property key name is not statically known.
+
+[source, syntax]
+----
+REMOVE n[key]
+----
+
+The dynamically calculated key must evaluate to a `STRING` value.
+This query creates a copy of every property on the nodes:
+
+
+.Query
+[source, cypher, indent=0]
+----
+MATCH (n) 
+WITH [k IN keys(n) WHERE k CONTAINS "Test" | k] as propertyKeys
+FOREACH (i IN propertyKeys | REMOVE k[i])
+RETURN n.name, keys(n);
+----
+
```

Comment:
I guess we should be clear that it gets all the keys of the node (singular) it is operating on, it isn't getting the keys of all the matched nodes (plural), but otherwise I think this good.

Could even mention it removes the corresponding property dynamically 😏 hehe but that is probably obvious 😅